.github/workflows/ci.yml

```
@@ -0,0 +1,17 @@
+name: CI
+
+on: push
```

Could you also enable it for pull requests?
```suggestion
on:
  push:
    branches: [master]
  pull_request:
    branches: [master]
```